external-crates/move/move-compiler/src/command_line/compiler.rs

```diff
@@ -214,6 +214,16 @@ impl<'a> Compiler<'a> {
     ) -> anyhow::Result<(
         FilesSourceText,
         Result<(CommentMap, SteppedCompiler<'a, TARGET>), Diagnostics>,
+    )> {
+        self.run_with_warning_filter(None)
+    }
+
+    pub fn run_with_warning_filter<const TARGET: Pass>(
+        self,
+        warning_filter: Option<WarningFilters>,
+    ) -> anyhow::Result<(
+        FilesSourceText,
+        Result<(CommentMap, SteppedCompiler<'a, TARGET>), Diagnostics>,
```

Comment:
Ah, neat! Makes sense from a command line perspective. 

But I think this would be more inline with the rest of the API if you did something like `fn add_warning_filter(s)` with the other `add_` and `set_` functions above. Then we won't need a second run function